src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -290,7 +310,7 @@ Type[] IInvocationBinder.GetParameterTypes(string methodName)
             Type[] types;
             if (!_paramTypes.TryGetValue(methodName, out types))
             {
-                throw new InvalidOperationException($"The hub method '{methodName}' could not be resolved.");
+                return Type.EmptyTypes;
```

Comment:
This does not seem right. If you are using non-self descriptive format (like protobuf) you don't know what to read and you will mess up the stream.